bitnami/grafana-operator/templates/grafana.yaml

```diff
@@ -188,24 +181,11 @@ spec:
           secretName: {{ .Values.grafana.ingress.tlsSecret }}
       {{- end }}
     metadata:
-      labels: {{- include "common.labels.standard" . | nindent 8 }}
-        {{- if .Values.commonLabels }}
-        {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 8 }}
-        {{- end }}
-        {{- if .Values.grafana.labels }}
-        {{- include "common.tplvalues.render" ( dict "value" .Values.grafana.labels "context" $ ) | nindent 8 }}
-        {{- end }}
-        {{- if .Values.grafana.ingress.labels }}
-        {{- include "common.tplvalues.render" ( dict "value" .Values.grafana.ingress.labels "context" $ ) | nindent 8 }}
-        {{- end }}
-      {{- if or .Values.commonAnnotations .Values.grafana.ingress.annotations }}
-      annotations:
-        {{- if .Values.commonAnnotations }}
-        {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 8 }}
-        {{- end }}
-        {{- if .Values.grafana.ingress.annotations }}
-        {{- include "common.tplvalues.render" ( dict "value" .Values.grafana.ingress.annotations "context" $ ) | nindent 8 }}
-        {{- end }}
+      {{- $labels := merge .Values.grafana.ingress.labels .Values.grafana.labels .Values.commonLabels }}
+      labels: {{- include "common.labels.standard" ( dict "customLabels" $labels "context" $ ) | nindent 8 }}
+      {{- if or .Values.commonAnnotations .Values.grafana.persistence.annotations }}
+      {{- $annotations := merge .Values.grafana.persistence.annotations .Values.commonAnnotations }}
```

Comment:
@juan131 Unfortunately I missed another issue in this MR.  This seems to be a simple copy & paste error. You are using the persistence annotations instead of the ingress annotations for the ingress here.

It seems also someone mentioned this in #18249 (Link to comment: https://github.com/bitnami/charts/issues/18249#issuecomment-1694767191).

Do you think you can fix this issue in a another MR?

Reply:
Good catch @BernhardGruen, I just created https://github.com/bitnami/charts/pull/18973 to address it